patternfly/patternfly-elements
# feat: react wrappers

## What I did

1. add a function to pfe-tools which takes a custom elements manifest and and output directory and generates react wrappers from the cem
2. run that function for our cem on each build, and gitignore the results
3. add those exports to the elements package under the `/react` dir

## Testing Instructions

1. `npm run build`
2. `npm link -w @patternfly/elements`
3. `git clone https://github.com/RedHat-UX/pfe-react-wrappers-demo.git`
4. `cd pfe-react-wrappers-demo`
5. `npm ci`
6. `npm link @patternfly/elements`
7. `npm run dev`
8. see if you have a reaction (side effects include rash, irritability, loss of sleep, lowered libido, and increased visibility to conference organizers and recruiters)

## Notes to Reviewers

1. sigh
